src/main/flight/imu.h

```diff
@@ -43,12 +43,12 @@ extern attitudeEulerAngles_t attitude;
 
 typedef struct imuConfig_s {
     // IMU configuration
-    uint8_t pid_process_denom;              // Processing denominator for PID controller vs gyro sampling rate
+    uint16_t max_angle_inclination;         // max inclination allowed in angle (level) mode. default 500 (50 degrees).
+    uint8_t small_angle;                    // Angle used for mag hold threshold.
     uint16_t dcm_kp;                        // DCM filter proportional gain ( x 10000)
     uint16_t dcm_ki;                        // DCM filter integral gain ( x 10000)
-    uint8_t small_angle;                    // Angle used for mag hold threshold.
-    uint16_t max_angle_inclination;         // max inclination allowed in angle (level) mode. default 500 (50 degrees).
     uint8_t gyro_sync;                      // Enable interrupt based loop
```

Comment:
As argued before, and as @hydra agreed in separate PR, gyro related config should be ing `gyroConfig_t`. So `gyro_sync` and `gyro_sample_hz` should be in `gyroConfig_t`.

`pid_process_denom` has nothing to do with the IMU and should not be hear. I think the best place for it is in  `pidProfile_t` - it is really a parameter of the PID. For example, if it changes, the PID values themselves will probably have to change.